arch/risc-v/src/esp32c3/esp32c3_wireless.c

```diff
@@ -29,9 +29,17 @@
 
 #include "hardware/esp32c3_soc.h"
 #include "hardware/esp32c3_syscon.h"
+#include "esp32c3_wireless.h"
 #include "esp32c3.h"
 #include "espidf_wifi.h"
 
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define MAC_ADDR0_REG (DR_REG_EFUSE_BASE + 0x044)
+#define MAC_ADDR1_REG (DR_REG_EFUSE_BASE + 0x048)
```

Comment:
```suggestion
#define MAC_ADDR0_REG EFUSE_RD_MAC_SPI_SYS_0_REG
#define MAC_ADDR1_REG EFUSE_RD_MAC_SPI_SYS_1_REG
```
It is best to reuse the definitions from https://github.com/apache/incubator-nuttx/blob/master/arch/risc-v/src/esp32c3/hardware/esp32c3_efuse.h#L815

Reply:
done